src/core/order_manager.py

```diff
@@ -6,7 +6,7 @@
 class OrderManager:
     def __init__(self):
         self._order_status_to_order_queue_map = {
-            "order_placed": Queue(maxsize=0),
+            "new_order": Queue(maxsize=0),
             "in_process": Queue(maxsize=0),
```

Comment:
Please set a max limit of orders to handle by each queue and also make sure to avoid queue overflow

Reply:
One suggestion could be adding the logic right before enqueue

Reply:
done.